apps/explorer/lib/explorer/chain/import/runner/zilliqa/zrc2/token_adapters.ex

```diff
@@ -0,0 +1,79 @@
+defmodule Explorer.Chain.Import.Runner.Zilliqa.Zrc2.TokenAdapters do
+  @moduledoc """
+  Bulk imports `t:Explorer.Chain.Zilliqa.Zrc2.TokenAdapter.t/0`.
+  """
+
+  require Ecto.Query
+
+  alias Ecto.{Changeset, Multi, Repo}
+  alias Explorer.Chain.Import
+  alias Explorer.Chain.Zilliqa.Zrc2.TokenAdapter
+  alias Explorer.Prometheus.Instrumenter
+
+  @behaviour Import.Runner
+
+  # milliseconds
+  @timeout 60_000
+
+  @type imported :: [TokenAdapter.t()]
+
+  @impl Import.Runner
+  def ecto_schema_module, do: TokenAdapter
+
+  @impl Import.Runner
+  def option_key, do: :zrc2_token_adapters
```

Comment:
We usually follow the convention that chain type related option keys start with the chain type name. Thus, can we change the option key to include the chain name?  Like this:

```suggestion
    def option_key, do: :zilliqa_zrc2_token_adapters
```